cypress/support/commands.ts

```diff
@@ -9,6 +9,15 @@ import { Organization } from '../../frontend/awx/interfaces/Organization';
 import { Project } from '../../frontend/awx/interfaces/Project';
 import { Team } from '../../frontend/awx/interfaces/Team';
 import { User } from '../../frontend/awx/interfaces/User';
+<<<<<<< HEAD
```

Comment:
Looks like some random merge editor code snuck in here.

Reply:
Thanks! Addressed 👍